zulipterminal/ui_tools/views.py

```diff
@@ -1330,6 +1330,19 @@ def __init__(self, controller: Any, stream_id: int) -> None:
         ]
 
         total_members = len(stream["subscribers"])
+        Stream_id = stream["stream_id"]
+        type_of_stream = "Private" if stream["invite_only"] is True else "Public"
+        stream_posting_policy = {
+            1: "Any user can post",
+            2: "Only administrators can post",
+            3: "Only full members can post",
+            4: "Only moderators can post",
+        }
+        posting_policy = stream_posting_policy[stream["stream_post_policy"]]
+        availability_of_history = (
+            "Yes" if stream["history_public_to_subscribers"] else "No"
+        )
+        user_role = "Stream administrator" if stream["role"] == 20 else "Subscriber"
```

Comment:
I think we decided to leave the subscriber role for now?

Reply:
Removed that.